external/emulation/emulation/_emulate/microphysics.py

```diff
@@ -73,6 +79,18 @@ def always_emulator(state: FortranState, emulator: FortranState):
     return emulator
 
 
+def predict(model: tf.keras.Model, state: FortranState) -> FortranState:
```

Comment:
```suggestion
def _predict(model: tf.keras.Model, state: FortranState) -> FortranState:
```
Suggest keeping this private.